sandbox/pages/SandboxPop.vue

```diff
@@ -0,0 +1,295 @@
+<template>
+  <SandboxLayout
+    :links="inject('app-links', [])"
+    title="KStepper"
+  >
+    <div class="kstepper-sandbox">
```

Comment:
A few instances in the sandbox still reference KStepper

Reply:
Fixed